src/index.html

```diff
@@ -5,8 +5,449 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+    rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+          class="header__logo-moyo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Samsung</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Smartphones</a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Gadgets</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Tablets</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Photo</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog catalog--position">
+        <section class="card card--position" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="product-item-imac27"
+          >
+
+          <div class="card__description">
+            <h5 class="card__description-title">
```

Comment:
The card title should be wrapped within a 'h2' element instead of 'h5' to follow the correct hierarchy of headings. This is also important for accessibility and SEO.